google/cloud/storage/internal/read_object_range_request.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/read_object_range_request.h"
+#include <sstream>
+
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+ReadObjectRangeResponse ReadObjectRangeResponse::FromHttpResponse(
+    HttpResponse&& response) {
+  auto loc = response.headers.find(std::string("content-range"));
```

Comment:
What if this gets returned as "Content-Range"? I imagine the case difference would result in this `find()` call failing to find this field, although I haven't tested it.

Reply:
It would, I think we need a case-insensitive map for the headers: #750.